js/index.js

```diff
@@ -71,7 +71,7 @@ function setDownloadURI() {
 function updateImage() {
   var query_string = generateQueryString();
   var preview = document.getElementById("preview");
-  preview.src = "../img/coderdojocertificate.svg" + query_string;
```

Comment:
Oh I thought Javascript executes relative to where the JS file is located. Now I understand, thanks